### ISSUE TITLE
Problema con seed T4 Parte 1

### ISSUE THREAD

**Issue by BrunoFarfan** [IIC2613-Inteligencia-Artificial-2022-1/Syllabus#108]
Hola!,

Creo que no me está funcionando bien el tema de la _seed_ ya que cada vez que ejecuto el código, no me dan los mismo resultados.

Lo que hice fue poner una celda al principio con:

```
import numpy as np

np.random.seed(20642300)
```

¿Hice algo mal? ¿Necesito poner la _seed_ en **cada** celda?
(Con cada vez que ejecuto el código, me refiero a cada vez que le pongo **"Ejecutar todo"** en VSCode, por ejemplo)

**Comment by FeBalla** [comment]
Hola!
Esto es normal, ya que la _seed_ de numpy, si bien controla la mayoría de los casos, deja algunas funciones/métodos sin abarcar. En este caso, lo que deberías hacer es identificar el punto en el que se están haciendo los cálculos aleatorios (puede ser la división con `train_test_split`) y darle el parámetro `random_state` con la misma semilla.
